### PR TITLE
[release-v1.56] Allow Deployed CDI to get out of Error Phase

### PR DESCRIPTION
**Pull request by mhenriks**
**What this PR does / why we need it**:

manual backport of #3080

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

https://issues.redhat.com/browse/CNV-38363

**Special notes for your reviewer**:

**Release note**:

```release-note
BugFix:  Allow Deployed CDI to get out of Error Phase
```

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -360,15 +360,15 @@ rpm(
 )
 
 rpm(
-    name = "buildah-1__1.31.3-1.el9.aarch64",
-    sha256 = "75cee8238006cd2ad42054a66cb93a36d429c86d661d293a632c19acf87b24a9",
-    urls = ["http://mirror.stream.centos.org/9-stream/AppStream/aarch64/os/Packages/buildah-1.31.3-1.el9.aarch64.rpm"],
+    name = "buildah-2__1.33.5-1.el9.aarch64",
```
Comment (akalenyu) [inline WORKSPACE:363]: how come these are needed?

Comment (mhenriks) [reply WORKSPACE:363]: was getting 404s on rpms so did `make rpm-deps`

**Comment by mhenriks** [comment]
@akalenyu rebased on #3111 but still had an issue with crun.  Unfortunately we don't have our own mirror for kubevirt/kubevirt

**Comment by akalenyu** [comment]
> @akalenyu rebased on #3111 but still had an issue with crun. Unfortunately we don't have our own mirror for kubevirt/kubevirt

Guess you mean https://github.com/kubevirt/containerized-data-importer/pull/3113, yeah, we will have to wait for an infra based solution (https://github.com/kubevirt/project-infra/issues/3097)

/approve
/lgtm

**Pull request merged**